app/pdfs/passengers_PDF.rb

```diff
@@ -0,0 +1,34 @@
+require 'prawn/table'
+
+class PassengersPDF < Prawn::Document
+  def initialize(passengers, filters)
+    super(page_layout: :landscape, page_size: "A3" )
+    content_width = bounds.width - 10
+    header(filters)
+    mycooltable(passengers)
+  end
+
+  def mycooltable(passengers)
```

Comment:
Did you want to keep the name `mycooltable` for this method? Hint: no

Reply:
lol yeah i'll change it up